testing.js

```diff
@@ -51,7 +51,7 @@ export class Shescape {
 
   quote(arg) {
     if (this.shell === false) {
-      throw new Error();
+      throw new Error("Can't quote when shell is false");
```

Comment:
The integration (and unit?) tests need to be updated to better test the exact behavior per the [failing mutation testing run for integration tests](https://github.com/ericcornelissen/shescape/actions/runs/6603451939/job/17936594583).

Reply:
Resolved with 8fc90494ef062593e0109717ea5e3075186aef61